EssentialFeed/EssentialFeediOS/Image Comments UI/Controllers/ImageCommentCellController.swift

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeed
+
+public final class ImageCommentCell: UITableViewCell {
+	@IBOutlet private(set) public weak var usernameLabel: UILabel!
+	@IBOutlet private(set) public weak var dateLabel: UILabel!
+	@IBOutlet private(set) public weak var messageLabel: UILabel!
+}
+
+public final class ImageCommentCellController: NSObject {
+	private let viewModel: ImageCommentViewModel
+	private var cell: ImageCommentCell?
+
+	public init(viewModel: ImageCommentViewModel) {
+		self.viewModel = viewModel
+	}
+}
+
+extension ImageCommentCellController: UITableViewDataSource {
+	public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		1
+	}
+
+	public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		cell = tableView.dequeueReusableCell()
+		cell?.usernameLabel.text = viewModel.username
+		cell?.dateLabel.text = viewModel.createdAt
+		cell?.messageLabel.text = viewModel.message
+		return cell!
```

Comment:
Following my previous message, this class doesn't need to hold a strong reference to the cell. It can just create the cell and return it - `UITableView` will hold a strong reference to it already as a subview:

```suggestion
		let cell: ImageCommentCell = tableView.dequeueReusableCell()
		cell.usernameLabel.text = viewModel.username
		cell.dateLabel.text = viewModel.createdAt
		cell.messageLabel.text = viewModel.message
		return cell
```